tests/phpunit/tests/block-supports/duotone.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+/**
+ * Test the block WP_Duotone class.
+ *
+ * @package WordPress
+ */
+
+class WP_Duotone_Test extends WP_UnitTestCase {
+	/**
+	 * Cleans up CSS added to block-supports from duotone styles. We neeed to do this
+	 * in order to avoid impacting other tests.
+	 */
+	public static function wpTearDownAfterClass() {
+		WP_Style_Engine_CSS_Rules_Store::remove_all_stores();
+	}
+
+	public function test_render_duotone_support_preset() {
```

Comment:
We probably need a Docblock here :)

Reply:
Let me know if this is enough, thank you!